### PR TITLE
Hospitals v2

### PR DESCRIPTION
**Pull request by vpremk**
@bodhish plz verify the jsons

@karmakoder lets not merge until Bodhi see, want to make sure the jsons is correct

### CODE REVIEW

File: scraper/src/v2_json_downloader.py [hunk 1]
```diff
@@ -11,26 +14,49 @@ def generate_v2_file(url, data, name=""):
...
-        active_district_data_v2_data["data"].append(active_district_json)
+    if len(active_district_data_v2_data["data"]) == 0:
+        for info_type_json in json_array:
+            active_district_json = dict(ambulance="false", contact="false", district="", doctor="false", helpline="false",
+                                        hospitals="false", medicine="false", oxygen="false", state="")
+            active_district_json["state"] = info_type_json["state_name"]
+            active_district_json["district"] = info_type_json["district_name"]
+            active_district_json[info_type] = "true"
+            #if(info_type == "ambulance" && TODO_LOGIC_FOR_AMBULENCE_TRUE):  #Update the logic for ambulance to be true based on values in ambulance.json
+                #active_district_json["ambulance"] = "true"
```
Comment (bodhish) [inline scraper/src/v2_json_downloader.py:33]: Is this required?

**Review by bodhish — APPROVED** [review]
LGTM

**Comment by bodhish** [comment]
We can override the conflicting files

File: scraper/src/v2_json_downloader.py [hunk 2]
```diff
@@ -11,26 +14,49 @@ def generate_v2_file(url, data, name=""):
...
-    active_district_data_v2_data = {
-        "data": []
-    }
+    s = set()
     json_array = json.loads(data)
-    for info_type_json in json_array:
-        active_district_json = dict(ambulance="false", contact="false", district="", doctor="false", helpline="false",
-                                    hospitals="false", medicine="false", oxygen="false", state="")
-        active_district_json["state"] = info_type_json["state_name"]
-        active_district_json["district"] = info_type_json["district_name"]
```
Comment (bodhish) [inline scraper/src/v2_json_downloader.py:27]: @vigneshhari do we need this now?
I think you have changed the output format to `state` and `district`

File: scraper/src/v2_json_downloader.py [hunk 3]
```diff
@@ -11,26 +14,49 @@ def generate_v2_file(url, data, name=""):
...
-        #if(info_type == "ambulance" && TODO_LOGIC_FOR_AMBULENCE_TRUE):  #Update the logic for ambulance to be true based on values in ambulance.json
-            #active_district_json["ambulance"] = "true"
-
-        active_district_data_v2_data["data"].append(active_district_json)
+    if len(active_district_data_v2_data["data"]) == 0:
+        for info_type_json in json_array:
+            active_district_json = dict(ambulance="false", contact="false", district="", doctor="false", helpline="false",
+                                        hospitals="false", medicine="false", oxygen="false", state="")
+            active_district_json["state"] = info_type_json["state_name"]
+            active_district_json["district"] = info_type_json["district_name"]
```
Comment (bodhish) [inline scraper/src/v2_json_downloader.py:30]:
```suggestion
            active_district_json["state"] = info_type_json["state"]
            active_district_json["district"] = info_type_json["district"]
```

@vigneshhari do we need this now?
I think you have changed the output format to `state` and `district`

**Review by bodhish — CHANGES_REQUESTED** [review]
@vandanabhandari could you replace the usage of `state_name` with `state` and `district_name` with `district`

@vigneshhari confirm if these changes where deployed

**Comment by vigneshhari** [comment]
Changed will be deployed at midnight

**Review by peeyush14goyal — CHANGES_REQUESTED** [review]

File: scraper/src/v2_json_downloader.py (hunk 3 above)
Comment (vandanabhandari) [reply scraper/src/v2_json_downloader.py]: https://life_data.coronasafe.network/hospital.json now has state and not state_name like it used to

Comment (vandanabhandari) [reply scraper/src/v2_json_downloader.py]: updated to have state and district now

File: data/hospital_clinic_centre_data_v2.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (vandanabhandari) [reply data/hospital_clinic_centre_data_v2.json:1]: I'm treating combination of state+district as id..

File: scraper/src/v2_json_downloader.py (hunk 1 above)
Comment (vandanabhandari) [reply scraper/src/v2_json_downloader.py]: removed brother [👍

File: data/hospital_clinic_centre_data_v2.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (vandanabhandari) [reply data/hospital_clinic_centre_data_v2.json:1]: added id for active_district_data_v2

**Comment by vandanabhandari** [comment]
@peeyush14goyal @bodhish @vigneshhari changes are updated . Thank you for reviewing amigos.

**Review by bodhish — APPROVED** [review]

File: data/hospital_clinic_centre_data_v2.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (vigneshhari) [reply data/hospital_clinic_centre_data_v2.json:1]: @vandanabhandari there is already a UUID present in each data object, the data will be de duplicated in the backend.

**Comment by shrihari689** [comment]
Huuurraaay 🎉 v2 in on 🔥

**Pull request merged**